src/rgw/rgw_user.cc

```diff
@@ -1850,6 +1861,12 @@ int RGWUser::check_op(RGWUserAdminOpState& op_state, std::string *err_msg)
     return -EINVAL;
   }
 
+  if (rgw_is_valid_tenant(op_id.tenant)) {
```

Comment:
true, it's probably worth mapping that to EINVAL.. but at least we'll print a concise error message